data/distro.yml

```diff
@@ -738,3 +738,32 @@
         <p>Check that in the Flatpak section the box is checked.</p>
         <p>Note: with this Flathub app search will be integrated in Discover, if you want to limit the app search to Flathub you can mark Flatpak as default by clicking on the star.</p>
     </ol>
+    
```

Comment:
Please, remove the four spaces (" ") on this line and replace them with just an empty line. Thanks!